todo-service/src/test/java/com/brtvsk/todoservice/TodoControllerIT.java

```diff
@@ -0,0 +1,265 @@
+package com.brtvsk.todoservice;
+
+import com.brtvsk.todoservice.model.dto.*;
```

Comment:
let's escape from wildcards on imports. Add a checkstyle rule

Reply:
Checkstyle rule is already applicable for the src code, but not for tests
Replaced all wildcard imports